modules/get-started/pages/whats-new.adoc

```diff
@@ -45,6 +45,18 @@ This release includes several updates to xref:get-started:licensing/overview.ado
 
 - *Unified license management in Redpanda Console*: You can now upload and apply a single license key for both Redpanda Console and the connected Redpanda cluster through the Redpanda Console UI. Any existing license key is overridden by the new one.
 
+== Backfill partitions
+
+When running xref:manage:cluster-maintenance/nodewise-partition-recovery.adoc[node-wise partition recovery], it's possible that there may be more recent data (a higher offset) available in Tiered Storage. Redpanda attempts to recover partition data from object storage, recovering the latest offset available for a partition in either storage tier (local or object storage). This allows for the maximum amount of data to be recovered in all cases, even for topics with a replication factor of 1, where no replicas remain in local storage.
+
+== Configure access to object storage with a KMS key
+
+Users on AWS or GCP with strict data compliance requirements can manage and store encryption keys separately from their cloud provider with a xref:manage:tiered-storage.adoc#configure-object-storage[customer-managed Key Management Service (KMS) key].
+
+== Intra-broker partition balancing
+
+xref:manage:cluster-maintenance/cluster-balancing.adoc#intra-broker-partition-balancing[Intra-broker partition balancing], which balances partitions across cores within a Redpanda broker, has moved out of beta and is supported for production clusters. 
+
```

Comment:
@mattschumpert Per the reviews for [this PR](https://github.com/redpanda-data/docs/pull/855#discussion_r1844920547) can we add a sentence to this section? Perhaps something like:

"Decreasing the number of CPU cores is now also supported for production clusters."

Reply:
@kbatuigas , sure, something like "Decreasing the number of CPUs assigned to an existing broker is also now supported.  Existing partition replicas are reassigned to the remaining CPU cores when the broker restarts with fewer available cores"